docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -9,52 +9,95 @@ priority: 2
 * Table of Contents
 {:toc}
 
-This guide describes how to run [Presto](https://prestodb.io/) with Alluxio, so
-that you can easily use Presto to query Hive tables stored in Alluxio's tiered storage.
+## Overview
+
+[Presto](https://prestodb.io/)
+is an open source distributed SQL query engine for running interactive analytic queries
+on data at a large scale.
+This guide describes how to run Presto to query Alluxio as a co-located distributed cache for
+Hive tables, where the data source can be AWS S3, Azure blob store, HDFS and many others.
+As a result, as long as a block is cached on one Alluxio worker node,
+it can also benefit other Presto nodes querying the same data.
 
 ## Prerequisites
 
-The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must use Java 8 Update 60 or higher (8u60+), 64-bit.
-Alluxio cluster should also be set up in accordance to these guides for either
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* [Deploy Presto](https://prestodb.io/docs/current/installation/deployment.html).
+This doc assumes and is tested with `presto-0.191`.
+* Alluxio cluster should also be set up according to these guides for either
```

Comment:
This might be too strong of a requirement - in real environments it's hard for users to follow one of these guides exactly, e.g. if they need to use Docker/Kerberos/Mesos/Yarn/etc. Is it possible to remove this requirement?

Reply:
I will just mention Alluxio service is running.